src/spotifyclient/runner.cpp

```diff
@@ -243,7 +242,29 @@ void SpotifyClient::Runner::onStarted()
 
 void SpotifyClient::Runner::onErrorOccurred(QProcess::ProcessError error)
 {
-	emit statusChanged(QString("Error %1").arg(error));
+	std::string_view message;
```

Comment:
Please note that this is still primarily a C++11 project, so `std::string_view` isn't available. It's something I'm planning to change for the next major release, as some configurations require C++17, and mixing versions is quite messy. However, in this case, I think `QStringLiteral` makes more sense, no point in working with C++ strings, when you only want to use it as a Qt string.